### PR TITLE
Assignment improvements

### PR DESCRIPTION
**Pull request by emanuelschuetze**
- Use ckeditor for assignment description field (e.g. for candidatures using html).
- Use 2 decimal places for percentage of voting results (motions/assignments)
- Allow css style attributes 'float' and 'padding'.
- Remove progress bar in election result slide to get more space (height) for candidate rows.

### CODE REVIEW

File: openslides/assignments/static/js/assignments/site.js [hunk 1]
```diff
@@ -128,9 +131,12 @@ angular.module('OpenSlidesApp.assignments.site', [
                 },
                 {
                     key: 'description',
-                    type: 'textarea',
+                    type: 'editor',
```
Comment (FinnStutzenstein) [inline openslides/assignments/static/js/assignments/site.js]: A server side check with `validate_html` is needed.

File: openslides/assignments/static/templates/assignments/assignment-detail.html [hunk 2]
```diff
@@ -77,10 +77,10 @@ <h3 translate>Phase</h3>
 <div class="details">
   <div ng-if="assignment.description">
     <h3 translate>Description</h3>
-    <div class="white-space-pre-line">{{ assignment.description }}</div>
+    <div ng-bind-html="assignment.description | trusted"></div>
   </div>
 
-  <div ng-if="assignment.phase !== 2">
+  <div ng-if="assignment.phase !== 2" style="clear: both">
```
Comment (FinnStutzenstein) [inline openslides/assignments/static/templates/assignments/assignment-detail.html]: Bootstrap has a class `clearfix` for this. Same for assignemt_slide.html

Comment (emanuelschuetze) [reply openslides/assignments/static/templates/assignments/assignment-detail.html]: Done

File: openslides/assignments/static/js/assignments/site.js (hunk 1 above)
Comment (emanuelschuetze) [reply openslides/assignments/static/js/assignments/site.js]: Done (in full and short serializer, right?)

**Comment by emanuelschuetze** [comment]
All done in separate review commit.
@FinnStutzenstein Please review again. Thx.

**Pull request merged**